test/Microsoft.AspNet.Mvc.Core.Test/Builder/MvcAreaRouteBuilderExtensionsTest.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Routing;
+using Microsoft.AspNet.Routing.Constraints;
+using Microsoft.AspNet.Routing.Template;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Builder
+{
+    public class MvcAreaRouteBuilderExtensionsTest
+    {
+        [Fact]
+        public void MapAreaRoute_Simple()
```

Comment:
> if you have nothing to say say nothing

All of these tests verify the most obvious possible thing. I don't see any reason to make the names more verbose.